app/controllers/application_controller.rb

```diff
@@ -1,2 +1,10 @@
 class ApplicationController < ActionController::Base
+  before_action :configure_permitted_parameters, if: :devise_controller?
+
+  protected
+    def configure_permitted_parameters
+      devise_parameter_sanitizer.permit(:sign_up, keys: [
```

Comment:
can you move this to a module and include it here, so the logic that setups Devise permitted parameters is self-contained?

Use `ActiveSupport::Concern`

```ruby
module SetupDevisePermitttedParams
  extend ActiveSupport::Concern
 
  
  ...
end
```